compiler/include/symbol.h

```diff
@@ -776,4 +776,33 @@ extern Symbol *gSingleVarAuxFields;
 extern std::map<FnSymbol*,int> ftableMap;
 extern std::vector<FnSymbol*> ftableVec;
 
+#define FUNC_NAME_MAX 256
+extern char llvmPrintIrName[FUNC_NAME_MAX+1];
+extern char llvmPrintIrStage[FUNC_NAME_MAX+1];
+
+//Names representations in LLVM IR and C generated code are
+//different from their names in AST. 'llvmPrintIrCName'
+//is place to keep name in LLVM IR and C version of
+//'llvmPrintIrName' variable.
+extern const char *llvmPrintIrCName;
+extern int llvmPrintIrStageNum;
+
+namespace llvmStageNum {
+enum { NOPRINT = 0,
```

Comment:
I'd consider making this a `typedef enum` and then using that type instead of int for llvmPrintIrStageNum.